pos_eater/static/src/xml/ActionpadWidget.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<templates id="template" xml:space="preserve">
+
+    <t
+        t-name="ActionpadWidget"
+        t-inherit="point_of_sale.ActionpadWidget"
+        t-inherit-mode="extension"
+        owl="1"
+    >
+        <xpath expr="//t[@t-esc='props.partner.name']" position="after">
+            <t t-if="props.partner.child_eater_ids">
+                <span class="eaters">
+                    <t t-esc="eaterNames" />
+                </span>
+            </t>
```

Comment:
So this is great, but if eaterNames is very long, the text disappears. See screenshot.

I do not know how to elegantly fix this.

![Capture d’écran du 2024-05-02 14-47-55](https://github.com/beescoop/Obeesdoo/assets/12065945/ad77a354-96c3-41d5-9999-87185bee0f3a)

Reply:
It is possible to get rid of the passing on the right. There's a `decentered` option for this. This doesn't solve the problem, but universally gives more space.